Recrutify/Recrutify.DataAccess/Repositories/ProjectRepository.cs

```diff
@@ -38,5 +38,14 @@ public Task IncrementCurrentApplicationsCountAsync(Guid id)
             var updateDefinition = Builders<Project>.Update.Inc(p => p.CurrentApplicationsCount, 1);
             return GetCollection().UpdateOneAsync(filter, updateDefinition);
         }
+
+        public Task<IEnumerable<Guid>> GetInterviewersIdsAsync(Guid projectid)
```

Comment:
in other methods param id, here is projectId...

Reply:
it's obvious that project

Reply:
+